OptiFineDoc/assets/minecraft/optifine/lang/zh_cn.lang

```diff
@@ -95,7 +95,11 @@ options.viewBobbing.tooltip.1=更真实的运动。
 options.viewBobbing.tooltip.2=当使用多级纹理贴图时，关闭它以获得最佳效果。
 
 options.guiScale.tooltip.1=界面尺寸
-options.guiScale.tooltip.2=更小的界面或许更快。
+options.guiScale.tooltip.2=  自动 - 最大尺寸
```

Comment:
I don't know your mean exactly...If you mean "why I indented it",please have a look on en_us.lang.